### PR TITLE
debugging dev deploy

### PR DESCRIPTION
**Pull request by esc**

### CODE REVIEW

File: .github/workflows/test_and_deploy.yml [hunk 1]
```diff
@@ -104,17 +104,15 @@ jobs:
...
-          auto-update-conda: true
-          auto-activate-base: false
-          activate-environment: test_env
       - name: Deploy to Anaconda Cloud
         shell: bash -l {0}
+        # workaround issues with setup-miniconda an anaconda-client
         run: |
+          source "$CONDA/etc/profile.d/conda.sh"
+          conda config --set always_yes yes --set changeps1 no
+          conda config --set always_yes yes --set changeps1 no
```
Comment (stuartarchibald) [inline .github/workflows/test_and_deploy.yml:113]: Is this needed 2x?

**Review by stuartarchibald — COMMENTED** [review]
Thanks for the patch. One question else looks good.

File: .github/workflows/test_and_deploy.yml (hunk 1 above)
Comment (esc) [reply .github/workflows/test_and_deploy.yml:113]: Good catch.

Comment (esc) [reply .github/workflows/test_and_deploy.yml]: fixed and pushed.

**Review by stuartarchibald — APPROVED** [review]
Thanks for the patch and fix.

**Pull request merged**